internal/endpoint/mqtt.go

```diff
@@ -3,12 +3,14 @@ package endpoint
 import (
 	"crypto/tls"
 	"crypto/x509"
+	"crypto/rand"
```

Comment:
The `crypto/rand` package is pretty slow vs `math/rand`. I'm concerned that the throughput for notifications might be gimped. Thoughts?

Reply:
The mqtt session is established only once per hook at the first event and remains open  for subsequent messages. But indeed cryptographic function is not needed for simple randomizer - the client id should be unique only among currently active clients